openedx/core/djangoapps/content/learning_sequences/migrations/0002_coursesectionsequence_inaccessible_after_due.py

```diff
@@ -1,18 +0,0 @@
-# Generated by Django 2.2.14 on 2020-07-09 06:01
```

Comment:
@ahmedbilal, why are you deleting a migration that has already been committed to the repository? You should only be adding migrations on top of the existing migrations and not delete the existing ones.